.dockerignore

```
@@ -0,0 +1,4 @@
+*.env.*
```

```suggestion
*.env*
```
You should make sure to ignore `.env`